### DISCUSSION TITLE
Show us your log!

### DISCUSSION THREAD

**Discussion by HB9HIL** [wavelog/wavelog#337] [Show and tell]
Post your public log here :)

**Comment by HB9HIL** [comment]
https://wavelog.hb9hil.org/visitor/log 👍😎

**Reply by R1BLH** [reply]
@HB9HIL Could you please advise how to hide the time of the QSO in the public dashboard view?

**Reply by HB9HIL** [reply]
Hello Michael

Go to your Wavelog instance and log yourself out. Then go to your public view: https://wavelog.cats-home.net/index.php/visitor/R1BLH

you will see that the time is only displayed if you are logged in. All other users do not see the QSO time. 

(You can enabled the QSO time also for not logged in users if you set `$config['show_time'] = FALSE;` to `TRUE` in `application/config/wavelog.php`). But I would not recommend that.

**Reply by wafer-li** [reply]
I wonder how to hide the map in public views.

The OpenStreet map has some political problems relate to China.

**Reply by HB9HIL** [reply]
> I wonder how to hide the map in public views.
> 
> The OpenStreet map has some political problems relate to China.

Soon possible https://github.com/wavelog/wavelog/pull/409

**Reply by R1BLH** [reply]
> you will see that the time is only displayed if you are logged in. All other users do not see the QSO time.

Oops... Mea culpa...
Thank you!!!  :)

**Reply by arguser** [reply]
Hello @HB9HIL, is it possible to count/show confirmations from QRZ on Public Logbooks? Also show lines and colours in map?

**Reply by HB9HIL** [reply]
@arguser You can create a new discussion in ideas :) https://github.com/wavelog/wavelog/discussions/categories/ideas

**Comment by R1BLH** [comment]
https://wavelog.cats-home.net/index.php/visitor/R1BLH

**Comment by antonov124** [comment]
404 Page Not Found
The page you requested was not found.    
,o(

https://domain/visitor/log
i dont use the index.php

**Reply by HB9HIL** [reply]
Replace `log` with the public slug you set in the Station Setup Page 

Enable the Visitor Site:

<img width="136" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/d2c9b60d-38eb-437e-8ac2-a2b680f31b6b">


And type in a visitor public "slug". What ever you like. The most people are using their callsign. But you also can use something else like "log" 

<img width="483" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/ffdbf971-c193-4028-859b-8aecd94dc1f7">

Then click save and visit `https://domain/visitor/[your_slug]` 
or you click on the second button here which brings you directly to this page.

<img width="222" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/fbeec59d-0a1d-4e7b-b8a0-ba587e50cf77">

**Reply by antonov124** [reply]
THX ;)

**Comment by spREGR** [comment]
[https://log.vu2tum.com/visitor/log](https://log.vu2tum.com/visitor/log)

**Reply by spREGR** [reply]
Hi, Can we have the Station Callsign on the header for visitor's log and OQRS request?

**Reply by phl0** [reply]
> Hi, Can we have the Station Callsign on the header for visitor's log and OQRS request?

Nope. Because those pages use logbooks and callsigns are associated with station locations rather than logbooks.

**Comment by dg9vh** [comment]
My public log is: https://log.dclnext.darc.de/visitor/dg9vh

**Comment by ChrisMakesHis** [comment]
My public log is here: [https://log.chrismakeshis.com/visitor/log](https://log.chrismakeshis.com/visitor/log)

**Comment by MikleBas** [comment]
My log [here ](https://log.ub3ayq.ru/index.php/visitor/ub3ayq)
:)
73!

**Comment by arguser** [comment]
My public log is here: https://log.escaping.network/visitor/lu5jpl

**Comment by PE1PQX** [comment]
My logs are listed here:
https://www.pe1pqx.eu/home/amateur-radio/mijn-logboeken (Sorry Dutch only, Google translate can/will help)
